.github/actions/latest-wrangler/main.py

```diff
@@ -0,0 +1,97 @@
+import os
+import sys
+import requests
+from distutils.util import strtobool
+from typing import Union
+from packaging.version import parse, Version
+
+if __name__ == "__main__":
+
+    # get inputs
+    package = os.environ["INPUT_PACKAGE"]
+    new_version = parse(os.environ["INPUT_NEW_VERSION"])
+    gh_token = os.environ["INPUT_GH_TOKEN"]
+    halt_on_missing = strtobool(os.environ.get("INPUT_HALT_ON_MISSING", "False"))
+
+    # get package metadata from github
+    package_request = requests.get(
+        f"https://api.github.com/orgs/dbt-labs/packages/container/{package}/versions",
+        auth=("", gh_token),
+    )
+    package_meta = package_request.json()
+
+    # Log info if we don't get a 200
+    if package_request.status_code != 200:
+        print(f"Call to GH API failed: {package_request.status_code} {package_meta['message']}")
+    
+    # Make an early exit if there is no matching package in github
+    if package_request.status_code == 404:
+        if halt_on_missing:
+            sys.exit(1)
+        else:
+            # everything is the latest if the package doesn't exist
+            print(f"::set-output name=latest::{True}")
+            print(f"::set-output name=minor_latest::{True}")
+            sys.exit(0)
+
+    # TODO: verify package meta is "correct"
```

Comment:
This one is not a leftover.  I think it would be wise to do a more formal validation of the state of the packages repo given we have certain expectations and if they aren't met it will break.  That said, I forgot to take out the todo and write a ticket as a follow up.  [Done now](https://github.com/dbt-labs/dbt-core/issues/4640) and removed from the code.